### ISSUE TITLE
Incorrect BaseDirectory and no WEBROOT_PATH Environment Variables are defined.

### ISSUE THREAD

**Issue by niemyjski** [Azure/azure-webjobs-sdk-script#1364]
I have a email notification job and after I set it up, I noticed that the email razor templates couldn't be resolved. This was because the `AppDomain.CurrentDomain.BaseDirectory` resolves to the app running under the program files folder. There was also no environment variable specified for the `wwwroot` directory so it made it quite hard to get to the right folder unless you specified a new environment variable.

The `WEBROOT...` environment variables should be set for function app web sites as well..

```
WEBROOT_PATH - The path to wwwroot.

WEBJOBS_TYPE - The type of the currently running WebJob.

WEBJOBS_NAME - The name of the currently running WebJob.
```

**Comment by fabiocav** [comment]
@niemyjski the current plan is to address this requirement through a context you can request as part of your function signature (much like you'd request the execution context).

Please see https://github.com/Azure/azure-webjobs-sdk-script/issues/803

**Comment by fabiocav** [comment]
Resolved by #1472
